pom.xml

```diff
@@ -141,13 +141,13 @@
     <dependency>
       <groupId>com.google.auto.value</groupId>
       <artifactId>auto-value</artifactId>
-      <version>1.3-rc2</version>
+      <version>1.4-rc2</version>
       <scope>provided</scope>
     </dependency>
     <dependency>
       <groupId>com.google.auto.service</groupId>
       <artifactId>auto-service</artifactId>
```

Comment:
nicest thing about this one is that annotations are now source retention, which makes the relationship even more explicit (compile-only dep)